### PR TITLE
MANATI-VM: Add missing config variables

### PR DESCRIPTION
**Pull request by rigoucr**
Steps to test:

- Update your config.yml file with the new variables added in the example.config.yml.
- Execute `vagrant up` and `vagrant  provision`
- Check if the changes of memory and cpu was applied to the VM.

### CODE REVIEW

File: example.config.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ---
 
+#VM Config
+vm_ram_memory: "2048"
```
Comment (kporras07) [inline example.config.yml]: @rigoucr could you name it only as "vm_ram"? Please :grin:

**Comment by kporras07** [comment]
It looks great! Only a really minor change :)

File: example.config.yml (hunk 1 above)
Comment (rigoucr) [reply example.config.yml]: yes :D

**Comment by kporras07** [comment]
Awesome! Assuming Travis passes; :shipit:

**Pull request merged**